design/concise-tidy-design.Rmd

```diff
@@ -0,0 +1,200 @@
+---
```

Comment:
Thanks @Puzzled-Face , can we please just have the one single `Rmd` file added in this PR? Currently I see the `html` too. I guess it is good that the `new.md` in snapshots was deleted though:

![image](https://github.com/Roche/crmPack/assets/1071227/9b9939a9-8e19-47cd-ae1f-ccbf7c3c9084)

Reply:
Added a `.gitignore` to `design` to ignore `.html` and `.pdf` files.

Reply:
I do not understand why `/design/.gitignore` is not removing the html file from the repo.